src/main/kotlin/org/jetbrains/research/testspark/display/ErrorMessageNormalizer.kt

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.research.testspark.display
+
+object ErrorMessageNormalizer {
+    /**
+     * Normalizes an error message by inserting "<br/>" tags after every 300 characters,
+     * except if there is already a "<br/>" tag within the blockSize characters.
+     * If the string length is a multiple of blockSize and the last character is a "<br/>" tag,
+     * it is removed from the result.
+     *
+     * @param error The error message to be normalized.
+     * @return The normalized error message.
+     */
+    fun normalize(error: String): String {
+        // set default parameters
```

Comment:
This function is a candidate for testing, as the implementation is somewhat tricky and is better ensured by unit tests.